sql/core/src/test/resources/sql-tests/inputs/datetime.sql

```diff
@@ -34,3 +34,5 @@ select date '2001-09-28' + 7;
 select 7 + date '2001-09-28';
 select date '2001-10-01' - 7;
 select date '2001-10-01' - date '2001-09-28';
+
+select make_date(2013, 7, 15);
```

Comment:
Please revert these `datetime.sql` and `datetime.sql.out`. Instead, please update `date.sql`.
- sql/core/src/test/resources/sql-tests/inputs/pgSQL/date.sql

You need to uncomment the followings.
```
-- select make_date(2013, 7, 15);
-- select make_date(-44, 3, 15);
-- select make_time(8, 20, 0.0);
-- should fail
-- select make_date(2013, 2, 30);
-- select make_date(2013, 13, 1);
-- select make_date(2013, 11, -1);
-- select make_time(10, 55, 100.1);
-- select make_time(24, 0, 2.1);
```